fenix/app/src/main/java/org/mozilla/fenix/settings/quicksettings/QuickSettingsFragmentStore.kt

```diff
@@ -142,16 +143,17 @@ class QuickSettingsFragmentStore(
             sessionId: String,
             websiteUrl: String,
             isTrackingProtectionEnabled: Boolean,
-            isCookieHandlingEnabled: Boolean,
+            cookieBannerUIMode: CookieBannerUIMode,
         ): ProtectionsState {
             return ProtectionsState(
                 tab = context.components.core.store.state.findTabOrCustomTab(sessionId),
                 url = websiteUrl,
```

Comment:
Is it possible to use the `url`parameter instead of introducing `siteDomainToReport`?

Reply:
Done